contributing.md

```diff
@@ -19,35 +19,41 @@ the right track.
 
 ### How to propose an RFC
 
-  1. Fork [the RFC repo](https://github.com/enarx/rfcs).
+  1. Fork [the RFC repo](https://github.com/enarx/rfcs). (cf. our 
+    [How to Contribute](https://github.com/enarx/enarx/wiki/How-to-contribute#git-environment-set-up)
+    document.)
   2. Choose a descriptive name for your RFC. Don't pick a number yet.
   3. Decide which parent folder is appropriate for your RFC: `/features` or 
     `/concepts`.
     If it is about a specific protocol or decorator or feature, its parent
     should be `/features`; if it is about a concept that will be used in many
     different features, its parent should be `/concepts`.
```

Comment:
are we currently doing this at all?  if not, should we remove?

Reply:
Forking repos, making changes, and then making PRs from our forks? Yes, we do that for all our contributions, not just RFCs.

If you're referring to the parent folders, I did that on the one RFC I've submitted thus far. I can see it being useful for organiz(s)ation as we scale up.

Reply:
I've gone over this a few times, i think if followed step by step, it's actually helpful to create the right files for the RFC. That being said, it is long and i think it could probably be simplified.